wordpress-mobile/WordPress-iOS
# [Stats Refresh] Period Overview card: filter tab bar sized properly

Fixes #n/a

This change:
- Fixes the Filter Tab Bar on the Overview Card to display properly on small devices. 
(As noted on https://github.com/wordpress-mobile/WordPress-iOS/pull/11204#issuecomment-470369452)
- Updates the Filter Tab Bar to emulate the Android implementation, i.e:
  - Centered tab labels.
  - Label in all caps.
  - Reduced label size to 12pt (just the label text, not the data part).
- This is unrelated, but it also removes the table footer from the detail list view. It was a 2 line change, so I threw it in.

To test:

---
- On devices of varying sizes, go to the Period > Overview card.
  - To note, the Filter Tab Bar was scrunched on the iPhone SE and XS.
- Verify the Filter Tab Bar is sized properly, i.e. the tab labels are two lines and not wrapped more. Examples:

iPhone SE:

<img width="350" alt="filter_tab_bar_SE" src="https://user-images.githubusercontent.com/1816888/54853118-8161b800-4cb4-11e9-849c-6a4a4d0e63ae.png">

iPad:

![ipad](https://user-images.githubusercontent.com/1816888/54853581-e1a52980-4cb5-11e9-93ca-e9ce7776b80a.png)

---
- Go to any "standard" details list (i.e. _not_ Latest Post Summary or Posting Activity).
- Scroll to the bottom of the list. Verify there is no longer a space between the table bottom and the tab bar.
